com.microsoft.mrtk.input/Simulation/InputSimulator.cs

```diff
@@ -228,10 +223,7 @@ public EyeGazeSimulationSettings EyeGazeSettings
         /// </remarks>
         private void EnableSimulatedEyeGaze()
         {
-            if (simulatedEyeGaze == null)
-            {
-                simulatedEyeGaze = new SimulatedEyeGaze();
-            }
+            simulatedEyeGaze ??= new SimulatedEyeGaze();
```

Comment:
Boy, wouldn't it be nice if we could use this syntax everywhere...

Reply:
It sure would!